### PR TITLE
Task-management feature spec

### PR DESCRIPTION
**Pull request by chaochaowu**
1. plus gem: capybara, launchy, database_cleaner, factory_bot
2. set default as Chrome and run test as GUI
3. set task-managements feature spec

ps. read commit to get more details :)

### CODE REVIEW

File: spec/factories/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :task do
+    user_id { user_id }
```
Comment (chain329) [inline spec/factories/tasks.rb]: 因為有關聯，可以改用`user`就好囉～

File: spec/features/task_management_spec.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        find("option[value='7']", text: 'July').select_option
+        find("option[value='12']", text: '12').select_option
+      end
+      fill_in 'task[description]', with: 'this is task test'
+      
+      click_button 'Create'
+      
+      expect(page).to have_content 'Create Success'
+      expect(Task.count).to eq 1
+      last_task = Task.last
```
Comment (chain329) [inline spec/features/task_management_spec.rb]: 這邊可以拉到上面去用，可改可不改，沒有一樣
```
let(:last_task) { Task.last }
```

File: spec/factories/tasks.rb (hunk 1 above)
Comment (chaochaowu) [reply spec/factories/tasks.rb]: 已更改～

File: spec/features/task_management_spec.rb (hunk 2 above)
Comment (chaochaowu) [reply spec/features/task_management_spec.rb]: 已更改～

**Pull request merged**